fdfgen/__init__.py

```diff
@@ -57,6 +59,9 @@ def handle_data_strings(fdf_data_strings, fields_hidden, fields_readonly):
 
 
 def handle_data_names(fdf_data_names, fields_hidden, fields_readonly):
+    if isinstance(fdf_data_names, dict):
+        fdf_data_names = fdf_dfdf_data_namesata_strings.items()
```

Comment:
I'm guessing that `fdf_dfdf_data_namesata_strings` is a typo and you meant `fdf_data_names.items()`.

I'm going to fix that and merge, but let me know if it's actually supposed to be something else.